kevm-pyk/src/kevm_pyk/kproj/evm-semantics/serialization.md

```diff
@@ -21,8 +21,8 @@ Address/Hash Helpers
 -   `keccak` serves as a wrapper around the `Keccak256` in `KRYPTO`.
 
 ```k
-    syntax Int ::= keccak ( Bytes ) [function, total, smtlib(smt_keccak)]
- // -------------------------------------------------------------------------
+    syntax Int ::= keccak ( Bytes ) [function, total, injective, smtlib(smt_keccak)]
+ // --------------------------------------------------------------------------------
```

Comment:
I guess we've largely decided that this is unsound and we shouldn't do it (and will take another approach): https://github.com/runtimeverification/kontrol/issues/165#issuecomment-1860803741.

So maybe we can just pull this out of this PR, so the other good lemmas here can be merged?

Reply:
Yes